.github/workflows/main.yml

```diff
@@ -13,7 +13,7 @@ on:
       - '.travis.yml'
       - '.github/workflows/quarkus.yml'
       - '**.md'
-  workflow_dispatch: []
+  workflow_dispatch:
```

Comment:
What's the point of an empty event?

Reply:
Thanks. A comment to explain that would be great.

Reply:
Done